src/main/java/NoSuchElementException.java

```diff
@@ -0,0 +1,5 @@
+public class NoSuchElementException extends RuntimeException {
+    public NoSuchElementException(String message) {
+        super(message);
+    }
+}
```

Comment:
I think its redundant to create a class of this exception. It can be just imported)